fips.conf

```diff
@@ -0,0 +1 @@
+machine private-ppa.launchpadcontent.net/ubuntu-advantage/pro-fips-updates login the-meulengracht password RVtd2H1Fvh4znwcZ7Nqn
```

Comment:
Oops?

Reply:
!!!!

Reply:
Removed it, deleting your comment to remove it from history